### ISSUE TITLE
checksums does not build any longer

### ISSUE THREAD

**Issue by phideg** [nabijaczleweli/checksums#18]
```
error[E0283]: type annotations required: cannot resolve `std::string::String: std::convert::AsRef<_>`
   --> /home/phil/.cargo/registry/src/github.com-1ecc6299db9ec823/pbr-0.3.1/src/pb.rs:348:64
    |
348 |                         base = base + repeat!(self.bar_current.as_ref(), curr_count - 1) +
    |                                       -------------------------^^^^^^------------------- in this macro invocation

error: aborting due to previous error

error: Could not compile `pbr`.
```

The issue can fixed with an update to version 1.0 of `pbr`.

**Comment by nabijaczleweli** [comment]
Released in [`v0.5.5`](https://github.com/nabijaczleweli/checksums/releases/tag/v0.5.5).

**Comment by phideg** [comment]
thx :+1:
